kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/LoadMultipleDocumentsFromFileExample.java

```diff
@@ -35,21 +35,14 @@ public static void main(String[] args) throws InterruptedException {
       master = args[0];
     }
 
-    Config config = new ConfigBuilder().build();
+    Config config = new ConfigBuilder().withMasterUrl(master).build();
     KubernetesClient client = new DefaultKubernetesClient(config);
 
     List<HasMetadata> list = client.load(LoadMultipleDocumentsFromFileExample.class.getResourceAsStream("/multiple-document-template.yml")).get();
     System.out.println("Found in file:" + list.size() + " items.");
     for (HasMetadata meta : list) {
       System.out.println(display(meta));
     }
-
-    list = client.load(LoadMultipleDocumentsFromFileExample.class.getResourceAsStream("/multiple-document-template.yml")).accept((Visitor<ObjectMetaBuilder>) item -> item.addToLabels("visitorkey", "visitorvalue")).get();
```

Comment:
Umm, Thanks for fixing this. Did you get any chance to find out why this section of code was throwing an error? I think that can be done as a separate issue also.

Reply:
this line should not be deleted - it will work if you turn the lambda into an anonymous class instead. casting should be also avoided since its error-prone with this library

Reply:
@pk044 : Thanks for the feedback, it worked!
@rohanKanojia : It was due to lambda for some reason.